actions/push-to-gar-docker/README.md

```diff
@@ -35,6 +35,13 @@ jobs:
           environment: "dev" # can be either dev/prod
 ```
 
+[Artifact Registry repositories can't container underscores][underscore-issue].
```

Comment:
**typo**: s/container/contain/

```suggestion
[Artifact Registry repositories can't contain underscores][underscore-issue].
```

Reply:
ah sorry for the auto-merge. I'll fix it

Reply:
oh you did already **NICE**